lib/storage/src/content_manager/mod.rs

```diff
@@ -31,7 +31,7 @@ pub mod consensus_ops {
     #[derive(Debug, Deserialize, Serialize, PartialEq, Eq, Hash, Clone)]
     pub enum ConsensusOperations {
         CollectionMeta(Box<CollectionMetaOperations>),
-        AddPeer(PeerId, String),
```

Comment:
It will break the serialization compatibility, won't it?

Reply:
Yes, it probably will. I can remove this change if we want to maintain compatibility